### DISCUSSION TITLE
Allow for disabling a badge's icon.

### DISCUSSION THREAD

**Discussion by tdejneka** [home-assistant/frontend#21792] [Dashboard]
The new badge design provides great flexibility for controlling its appearance. However, it makes the display of an icon mandatory.

Not all badge applications require an icon. For example, a row of six room temperatures doesn't benefit from displaying six thermometer icons. The icons simply occupy valuable screen real-estate (on a phone).  "Bedroom 21.5 C" is sufficient to convey the information. Other applications include humidity, light level, etc.

Despite its faults, the old badge design was narrower and allowed for displaying a single row of many values without wrapping to another row.

It would be appreciated if the new badge design allowed the icon to be disabled. The space it would normally occupy would be eliminated thereby permitting the badge's width to be narrower.

**Comment by ildar170975** [comment]
> It would be appreciated if the new badge design allowed the icon to be disabled

Agreed. Otherwise users have to use card-mod to hide an icon which is not convenient & native.

**Comment by c2bull** [comment]
Agree completely and sample badges are shown in this other discussion

https://github.com/home-assistant/frontend/discussions/21523

**Comment by karwosts** [comment]
Duplicate of #21523, has been answered.
